python/paddle/fluid/imperative/nn.py

```diff
@@ -44,14 +44,9 @@ def __init__(self,
         super(Conv2D, self).__init__(name_scope, dtype=dtype)
 
         # TODO(minqiyang): Move this to the top.
-        from ..layer_helper import LayerHelper
-        self._helper = LayerHelper(
-            self.full_name(),
-            param_attr=param_attr,
-            bias_attr=bias_attr,
-            dtype=dtype,
-            act=act)
-
+        self._helper.param_attr = param_attr
+        self._helper.bias_attr = bias_attr
+        self._helper.act = act
```

Comment:
hide _helper? and call create_parameter?

Reply:
done